src/data/local/github/tracks/track-2/steps/step-5.md

```diff
@@ -1,6 +1,6 @@
 # Step 5 - Rebasing your branch
 
-Here in Taqtile, before merging our revised (and approved) branches, we **rebase** this branch into the one we are attempting to merge. In this case, you'll probably rebase `docs/readme` to `develop`.
+Here in Taqtile, before merging our revised (and approved) branches, we **rebase** this branch into the one we are attempting to merge. In this case, you'll rebase `docs/readme` or `feature/setup` to `develop`, depending on which one was merged first.
 
 Too confusing? Here are some good links you can read about **rebase**:
 - [Git Rebasing](https://git-scm.com/book/en/v2/Git-Branching-Rebasing)
```

Comment:
Aqui tem um `feel free to ask to a Taqtiler if you're stuck in this step`?

Reply:
Tem logo abaixo